splitio/push/sse.py

```diff
@@ -219,26 +219,11 @@ async def start(self, url, extra_headers=None):  # pylint:disable=protected-acce
             await self._conn.close()
             self._conn = None  # clear so it can be started again
             _LOGGER.debug("Existing SSEClient")
-            return
+        return
 
-    async def shutdown(self):
+    def shutdown(self):
```

Comment:
what's the point of this? if it's just for the sake of interface compliance, i don't think it's worth it. either remove `shutdown()` from the interface spec or drop the interface altogether. sync & async implementations will never be interchangeable, so there's not much point in keeping them

Reply:
I do agree its not worth it, but all SDKs are following the same spec, if we want to keep consistency across all, we should follow the same interface just for that.